doc/ref/states/requisites.rst

```diff
@@ -423,6 +512,43 @@ The ``onfail`` requisite is applied in the same way as ``require`` as ``watch``:
 
 .. _Issue #22370: https://github.com/saltstack/salt/issues/22370
 
+.. _requisites-onfail:
```

Comment:
`_requisites-onfail` --> `_requisites-onfail_any`. (The previous wording is already used in this document.)